packages/astro/src/vite-plugin-fetch/index.ts

```diff
@@ -2,7 +2,7 @@ import type { Plugin } from '../core/vite';
 import MagicString from 'magic-string';
 
 // https://github.com/vitejs/vite/discussions/5109#discussioncomment-1450726
-function isSSR(options: undefined | boolean | { ssr: boolean }): boolean {
+function isSSR(options: undefined | boolean | { ssr?: boolean }): boolean {
```

Comment:
This should probably go away entirely in `2.7`. The updated types have a different signature than when we added this.